exampleLitActions/src/conditionalSigning.ts

```diff
@@ -0,0 +1,238 @@
+import { LitNodeClient } from '@lit-protocol/lit-node-client';
+import { LIT_RPC, LIT_NETWORK, LIT_ABILITY } from "@lit-protocol/constants";
+import {
+  createSiweMessage,
+  generateAuthSig,
+  LitActionResource,
+} from "@lit-protocol/auth-helpers";
+import { LitContracts } from "@lit-protocol/contracts-sdk";
+import * as ethers from "ethers";
+
+// Utility function to get environment variables
+const getEnv = (name: string): string => {
+  const value = process.env[name];
+  if (!value) {
+    console.warn(`Environment variable ${name} is not set`);
+  }
+  return value || "";
+};
```

Comment:
We should always define util methods in a `src/utils.ts` file. Our goal with the main code example file (this file) is to be as lean as possible, focusing only on the code that makes this code example unique. We don't want to use to have to filter out boilerplate code to find the code they'll want to copy/paste into their project